src/main/scala/wdl4s/cwl/CommandLineTool.scala

```diff
@@ -44,18 +42,18 @@ case class CommandInputEnumSchema(
 
 case class CommandInputArraySchema(
                                     items:
-                                      CwlType :+:
+                                    CwlType :+:
```

Comment:
I'm using IntelliJ auto-formatter and I guess so do you. Can you give me you configurations?